private-templates-service/server/src/app.ts

```diff
@@ -4,20 +4,20 @@ import passport from "./config/passport";
 import bodyParser from "body-parser";
 
 // import controllers
-import { TemplateServiceClient, ClientOptions, AzureADProvider, InMemoryDBProvider } from 'adaptivecards-templating-service';
+import { TemplateServiceClient, ClientOptions, AzureADProvider, InMemoryDBProvider } from '../../adaptivecards-templating-service';
```

Comment:
This change can be reverted since the import is described here: https://github.com/microsoft/adaptivecards-templates/tree/dev/private-templates-service/adaptivecards-templating-service#developer-instructions